apps/codelab/src/app/codelabs/codelabs.module.ts

```diff
@@ -1,23 +1,35 @@
 import { APP_INITIALIZER, NgModule } from '@angular/core';
 
+import { OverlayModule } from '@angular/cdk/overlay';
+
+import { monacoReady } from '@codelab/code-demos/src/lib/shared/monaco-config.service';
```

Comment:
import should be `@codelab/code-demos`
```suggestion
import { monacoReady } from '@codelab/code-demos';
```

Reply:
![image](https://user-images.githubusercontent.com/40883170/52910783-84304e00-32ad-11e9-8f89-b1c22a7b4852.png)

Is this ok?)

Reply:
It is necessary to add to the exports from the library